content/applications/finance/payment_acquirers/authorize.rst

```diff
@@ -115,6 +115,91 @@ order is canceled, you can click on *Void Transaction* to unlock the funds from
 .. note::
    With other payment acquirers, you can manage the capture in their own interfaces, not from Odoo.
 
+Authorize.Net statement export
+==============================
+
+.. _excel-file-template:
+
+.. admonition:: Template
+
+   You can find the Excel import template `here. <https://docs.google.com/spreadsheets/d/1CMVtBWLLVIrUpYA92paw-cL7-WdKLbaa/edit?usp=share_link&ouid=105295722917050444558&rtpof=true&sd=true>`_
+
+To import a statement, log into your Authorize.Net account, and go to :menuselection:`Account -->
+Statements --> eCheck.Net Settlement Statement`. Next, an export range must be defined. The range is
+defined by an **opening** and **closing** batch settlement, with all transactions contained
+in-between being the subject of the export. Select all transactions contained within the desired
+range, and copy/paste them into the :guilabel:`Report 1 Download` sheet of the :ref:`Excel sheet
+<excel-file-template>`.
+
+.. image:: authorize/authorize-report1.png
+   :align: center
+   :alt: Select all transactions for the desired range on Authorize.net, and copy them into
+         'report1' of the Excel template.
+
+.. example::
+   .. image:: authorize/authorize-settlement-batch.png
+      :align: center
+      :alt: Settlement batch of the an Authorize.Net statement
+
+   In this case, the first batch (01/01/2021) of the year belongs to the settlement of the
+   31/12/2020, so the **opening** settlement is from the 31/12/2020.
+
+Once the data has been pasted into the :guilabel:`Report 1 Download` sheet, go to
+:menuselection:`Authorize.net --> Transaction Search --> Search for a Transaction`, enter the
+previously used range of batch settlement dates, and click :guilabel:`Search`.
+
+When the list has been generated, click :guilabel:`Download to File`. In the pop-up window, select
+:guilabel:`Expanded Fields with CAVV Response/Comma Separated`, enable :guilabel:`Include Column
+Headings`, and finally click :guilabel:`Submit`. Open the text file, select :guilabel:`All`, copy
+the data and paste it into the :guilabel:`Report 2 Download` sheet of the Excel file.
+
+Transit lines are automatically filled-in and updated in the :guilabel:`transit for report 1` and
+:guilabel:`transit for report 2` sheets of the Excel file. Make sure all entries are present, and if
+not, copy the **formula** from previous filled-in lines of :guilabel:`transit for report 1` or
+:guilabel:`2` and paste it into the empty lines.
+
+.. important::
+   To get the same closing balance, do *not* remove any line from the Excel sheets.
+
+Import into Odoo
+----------------
+
+To import the data into Odoo, go to the Excel file, copy the data from the :guilabel:`transit
+report 2` sheet and **paste special** only the **values** in the :guilabel:`Odoo Import to CSV`
```

Comment:
Customers might use Google Sheets instead of Excel, and therefore the UI element might be different from software to software. I think it's better to bolden the concept rather than the UI element, as the concept is common to both.